tests/e2e/scenarios/test_reborn_responses_api.py

```
@@ -179,9 +179,18 @@ async def reborn_responses_client(reborn_responses_server):
         yield client
 
 
-async def create_response(client: httpx.AsyncClient, **payload) -> dict:
+async def create_response(
+    client: httpx.AsyncClient, path: str = "/v1/responses", **payload
```

Ruff (0.15.20)

[warning] Missing type annotation for `**payload`

(ANN003)